pxf/pxf-api/src/test/java/org/apache/hawq/pxf/api/FilterParserTest.java

```diff
@@ -215,6 +215,10 @@ public void parseColumnOnLeft() throws Exception {
         filter = "a1c2o7";
         op = Operation.HDOP_AND;
         runParseOneOperation("this filter was build from HDOP_AND", filter, op);
+        
+        filter = "a1c2o8";
+        op = Operation.HDOP_LIKE;
+        runParseOneOperation("this filter was build from HDOP_LIKE", filter, op);
```

Comment:
This reference to the previous code _`runParseOneOperation("this filter was build from HDOP_AND", filter, op)`_ ,  has been corrected.